cleaning/big_clean.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+tmp=$(mktemp -d)
+tar --gunzip -xf $1 -C $tmp
+here=$(pwd)
+cd $tmp 
+grep -rq 'DELETE ME!' . | xargs rm
+tar -czf $(basename cleaned_$1) .  
```

Comment:
Your use of `.` here is why your cleaning tests fail. It causes you to get `./little_dir` instead of `little_dir` in your final archive.